lib/async_hooks.js

```diff
@@ -4,6 +4,7 @@ const {
   NumberIsSafeInteger,
   ReflectApply,
   Symbol,
+  Map,
```

Comment:
Please keep the imports in alphabetical order.

Reply:
Fixed (commit pending).